src/bulma/parts/FormSection.vue

```diff
@@ -1,22 +1,25 @@
 <template>
-    <div class="columns is-multiline has-margin-bottom-large">
+    <div>
         <div class="column is-12"
             v-if="section.divider">
             <divider class="has-margin-bottom-medium"
                 :title="i18n(section.title)"
                 :placement="state.data.dividerTitlePlacement"/>
         </div>
-        <div v-for="field in sectionFields(section)"
-            class="column"
-            :class="section.columns !== 'custom'
-                ? columnSize(section.columns)
-                : `is-${field.column}`"
-            :key="field.name">
-            <slot :name="field.name"
-                v-if="field.meta.custom"/>
-            <form-field :field="field"
-                v-on="$listeners"
-                v-else/>
+        <div class="columns has-margin-bottom-large is-multiline"
+            v-for="(fields, index) in fieldChunks(section)"
+            :key="`chunk-${index}`">
+            <div v-for="field in fields"
+                class="column"
+                :class="section.columns === 'custom' && `is-${field.column}`"
+                :key="field.name">
+                <template v-if="field.dummy"/>
+                <slot :name="field.name"
+                    v-else-if="field.meta.custom"/>
+                <form-field :field="field"
+                    v-on="$listeners"
+                    v-else/>
+            </div>
```

Comment:
please test this as template
```vue
<div class="columns has-margin-bottom-large is-multiline"
    v-for="row in sectionRows(section)"
    :key="`${section.id}-${row}`">
    <div class="column"
        v-for="column in rowColumns(section)"
        :class="columnClass(section, row * column - 1)"
        :key="row * column - 1">
        <template v-if="row * column - 1 < section.fields.length">
            <slot :name="field.name"
                v-if="field.meta.custom"/>
            <form-field :field="field"
                v-on="$listeners"
                v-else/>
        </template>
    </div>
</div>
```
 
and js
```js
sectionRows(columns, { fields }) {
    return columns === 'custom'
        ? 1
        : Math.ceil(fields.length / columns);
},
rowColumns(columns, { fields }) {
    return columns === 'custom' ? fields.length : columns;
},
columnClass(section, index) {
    return section.columns === 'custom'
        ? `is-${section.fields[index].column}`
        : null;
},
```

Reply:
![image](https://user-images.githubusercontent.com/10864136/107633572-7c79c900-6c7d-11eb-9f32-7d14c7dbaa1c.png)
branch result:
![image](https://user-images.githubusercontent.com/10864136/107634788-56edbf00-6c7f-11eb-8b01-d1672e00b9ac.png)

Reply:
ah right, so probably it should be something like

`(row - 1) * rowColumns(section) + column - 1` instead of `row * column - 1`

Reply:
![image](https://user-images.githubusercontent.com/10864136/107637711-d1204280-6c83-11eb-9b6a-360796f88c9c.png)